### PR TITLE
Replace MN logo with project logo

### LINKED ISSUE #51: Presentability: use generic PSM logo
The PSM should have a generic (non-state-specific) logo for dev/demo sandboxes, etc.

Note that this ticket also used to include another presentability concern: getting rid of the "lorem ipsum" nonsense text in the UI, e.g., on the login page.  However, that is now tracked in issue #108.

### PR DESCRIPTION
**Pull request by jasonaowen**
Refactor to extract an `@include`-able fragment that has the logo in it, then replace the old logo with the new logo.

Resolves #51 use generic PSM logo

### CODE REVIEW

File: psm-app/cms-web/WebContent/WEB-INF/pages/includes/logo.jsp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <a class="logo" href="#">
   <img
-    src="<c:url value="/i/logo.gif" />"
+    src="<c:url value="/i/psm-logo.png" />"
     alt="Medicaid Provider"
```
Comment (cecilia-donnelly) [inline psm-app/cms-web/WebContent/WEB-INF/pages/includes/logo.jsp]: Can we change the alt text to "PSM logo" or something similar?

Comment (jasonaowen) [inline psm-app/cms-web/WebContent/WEB-INF/pages/includes/logo.jsp]: Sure! How about "Provider Screening Module"?

Comment (cecilia-donnelly) [inline psm-app/cms-web/WebContent/WEB-INF/pages/includes/logo.jsp]: Sounds good to me!

Comment (jasonaowen) [inline psm-app/cms-web/WebContent/WEB-INF/pages/includes/logo.jsp]: Done!

**Comment by cecilia-donnelly** [comment]
Thanks, @jasonaowen!  Looks great. :+1:

**Pull request merged**